### DISCUSSION TITLE
Search part of field value instead of full field value

### DISCUSSION THREAD

**Discussion by sghriyadh** [Budibase/budibase#10707] [Help] [answered]
Hi

i have a search form which can search customer name if you type exact full name of customer.

i want to search customer using part of full name.
it is like using "like" filter but in search box

for example

if customers name are

John doe
Jason dompyre
Smith Jalfer

if i search "do" then it returns the rows containing John doe and Jason dompyre

if i search "Ja" then it returns the rows containing Jason dompyre and Smith Jalfer.

Can you guide how to achieve this in search box?

**Comment by ConorWebb96** [comment, marked as answer]
Hey @sghriyadh,

You can use the like filter combined with a search field to achieve this, to set this up quickly you can follow the instructions below;

1. Add a Table Block component, configure this however you desire.
2. Add the field you wish to apply the search on. 
3. Eject the block.
4. Look for the `Dataprovider` It should be the parent of the table. Click on it.
5. On the right side of your screen, you should then be able to see `Define filters` button. Click this.
6. Then change the type of filtering from `Starts with` to `Like`

I've included an example below of this being done from start to finish.
![changing filtering from starts with to like](https://github.com/Budibase/budibase/assets/126772285/774c0b80-36df-4b83-be55-0e27bdc6344b)

Hopefully this makes sense.
